modules/gapi/samples/infer_single_roi.cpp

```diff
@@ -124,6 +130,21 @@ GAPI_OCV_KERNEL(OCVBBoxes, BBoxes) {
     }
 };
 
+
+GAPI_OCV_KERNEL(OCVRescaleByROI, RescaleByROI) {
```

Comment:
What is the reason to rescale objects?

Don't we need to shift them (e.g. change x/y coordinates only) when projecting a detection from ROI to the parent image?